pulp_container/constants.py

```diff
@@ -71,3 +75,34 @@
 SIGNATURE_PAYLOAD_MAX_SIZE = 4 * MEGABYTE
 
 SIGNATURE_API_EXTENSION_VERSION = 2
+
+MANIFEST_TYPE = SimpleNamespace(
+    IMAGE="image",
+    BOOTABLE="bootable",
+    FLATPAK="flatpak",
+    HELM="helm",
+    COSIGN_SIGNATURE="cosign_signature",
+    COSIGN_ATTESTATION="cosign_attestation",
+    COSIGN_ATTESTATION_BUNDLE="cosign_attestation_bundle",
```

Comment:
I am unsure we should differentiate between these. What would be a typical use case for seeing a difference between the bundle and regular attestation? Please, elucidate further.